media/css/mozorg/contribute/contribute.scss

```diff
@@ -2,176 +2,396 @@
-// Contribution Video
+.contribute-banner {
+    padding-top: $layout-xl;
+    padding-bottom: $layout-md;
+    padding-left: $layout-md;
+    padding-right: $layout-md;
 
-.is-modern-browser .contribute-video {
-    display: none;
-}
+    @media #{$mq-lg} {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        min-height: 260px;
+        position: relative;
+        padding-top: 0;
+        padding-bottom: 0;
+    }
 
-.mzp-c-modal video {
-    display: block;
-    margin: 0 auto;
-    width: 100%;
+    .contribute-banner-content {
+        @media #{$mq-lg} {
+            margin-left: 200px;
+            width: 60%;
+            max-width: 495px;
+        }
+    }
+
+    .contribute-banner-cta {
+        flex: 1 0 auto;
+
+        @media #{$mq-lg} {
+            margin-left: $layout-xl;
+        }
+    }
 }
 
-//* -------------------------------------------------------------------------- */
-// Mozillian Stories Links
+.contribute-banner-gethelp {
+    background-image: url("/media/img/contribute/contribute-gethelp-banner-bg.jpg");
+    background-size: cover;
+    color: $color-white;
 
-.contribute-mozillian-list {
-    margin-top: $layout-lg;
+    @media #{$mq-lg} {
+        margin-bottom: $layout-lg;
+    }
 
-    .contribute-mozillian-item {
-        @include border-box;
-        @include text-body-lg;
-        margin-bottom: $spacing-lg;
-        padding: 0 $spacing-md;
+    .contribute-banner-gethelp-title {
+        font-family: Inter, X-LocaleSpecific, sans-serif;
+        @include font-size(24px);
         position: relative;
-        text-align: center;
+
+        @media #{$mq-lg} {
+            @include font-size(40px);
+        }
+
+        &:before {
+            content: "";
+            display: block;
+            background-image: url("/media/img/contribute/contribute-banner-gethelp-icon.svg");
+            position: absolute;
+            height: 40px;
+            width: 40px;
+            left: 0;
+            top: -65px;
+
+            @media #{$mq-lg} {
+                top: auto;
+                left: -80px;
+            }
+        }
+    }
+}
+
+.contribute-banner-problems {
+    background-image: url("/media/img/contribute/contribute-problems-bulb.svg");
+    background-size: 60px 150px;
+    background-position: 10% -10px;
+    background-repeat: no-repeat;
+
+    @media #{$mq-sm} {
+        background-size: 50px 150px;
+        background-position: 8% -10px;
     }
 
     @media #{$mq-lg} {
-        @include clearfix;
+        border: 4px solid $color-moz-light-gray;
+        background-image: url("/media/img/contribute/contribute-problems-bulb.svg"),
+            url("/media/img/contribute/contribute-problems-fly.svg");
+        background-size: 86px 241px, 33%;
+        background-position: 10% -45px, 96% 88%;
+        background-repeat: no-repeat;
+    }
+
+    .contribute-banner-problems-title {
+        @include font-size(24px);
```

Comment:
Unfortunately it looks like the @include text-display-md mixin renders the font at 40px and 2.5rem instead of the 24px that is outlined in the design.